package.json

```diff
@@ -1,47 +1,47 @@
 {
-  "name": "concierge-bot",
-  "version": "4.1.4",
-  "description": "Extensible general purpose chat bot.",
-  "main": "main.js",
-  "dependencies": {
-    "babel-plugin-transform-strict-mode": "^6.3.13",
-    "babel-polyfill": "^6.3.14",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-register": "^6.3.13",
-    "babylon": "^6.8.0",
-    "coffee-script": "^1.10.0",
-    "colors": "^1.1.2",
-    "figlet": "^1.1.1",
-    "scoped-http-client": "^0.11.0",
-    "winston": "^2.3.1"
-  },
-  "devDependencies": {
-    "chai": "^3.5.0",
-    "grunt": "^1.0.1",
-    "grunt-contrib-watch": "^1.0.0",
-    "grunt-mocha-test": "^0.13.2",
-    "grunt-run": "^0.6.0",
-    "mocha": "^3.0.2"
-  },
-  "scripts": {
-    "test": "grunt test",
-    "start": "node main.js",
-    "debug": "node main.js --debug --timestamp"
-  },
-  "engines": {
-    "node": "6.2.1"
-  },
-  "bin": {
-    "concierge": "./main.js"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/concierge/Concierge.git"
-  },
-  "author": "Matthew Knox",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/concierge/Concierge/issues"
-  },
-  "homepage": "https://github.com/Concierge/Concierge"
-}
+    "name": "concierge-bot",
```

Comment:
I do like 4 spaces, but any reason why we are fixing an unrelated issue in this PR?